setup.cfg

```diff
@@ -49,10 +49,10 @@ python_requires >= 3.9
 # For more information, check out https://semver.org/.
 install_requires =
     importlib-metadata; python_version<"3.8"
-    aiohttp
-    pandas
+    aiohttp<=3.9.1
+    pandas>=2.1.4
```

Comment:
For a minimum requirement, an inline comment is helpful, too.

Reply:
This comes from Victor's work again. Should I ask him why he decided on this?